packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -286,15 +279,29 @@ const cfg = (options?: Options): Config => {
           loader: "loader 1.25s infinite ease-in-out",
           pulse: "pulse 1.5s infinite",
         },
-        textColor: Object.entries(getForegroundColors(tokens)).reduce((acc, [key, value]) => {
-          const name = key.replace("text-", "");
-          return { ...acc, [name]: value };
-        }, {}),
-        backgroundColor: getBackgroundColors(tokens),
+        textColor: {
+          ...Object.entries(getForegroundColors(tokens)).reduce((acc, [key, value]) => {
+            const name = key.replace("text-", "");
+            return { ...acc, [name]: value };
+          }, {}),
+          // overrides for specific WL
```

Comment:
Are these overrides? Or are these just class definitions? What are we overriding?

"overrides for specific WL" is also a bit vague. What speficic?

Reply:
Idk what specific, as the document does not contain tokens per each WL, it just mentions which are used for overrides. Anyway, I made them at least available. Imo we shouldn't even allow usage of this tokens, as the WL need only the palette, but this hacks are used because they have wrongly defined their palettes or not constrast enough.